### ISSUE TITLE
Questions with Detail Viz Type Poorly Formatted on Dashboards

### ISSUE THREAD

**Issue by andresrecalde** [metabase/metabase#51557]
### Describe the bug

When adding a Question with a Detail viz type to a dashboard, the whitespacing does not work as expected. It keeps an unnecessary amount of right-padded space; when the size is made smaller it comes at the expense of column names being formatted to multiple lines and ultimately truncated. This leads to requiring dashboards with a bunch of whitespace between questions to make sure names aren't cut out.

Example when fully across a large dashboard
<img width="1348" alt="Image" src="https://github.com/user-attachments/assets/5694217b-6009-45a0-9823-1bf1b9fa2134" />

Column names become multi-line when there is plenty of space 
<img width="1378" alt="Image" src="https://github.com/user-attachments/assets/8e6be2b7-7717-4574-a2ff-d153cd4e40c4" />

### To Reproduce

1. Go to any dashboard
2. Add a question with a Detail type
3. Change the question size to anything, the additional whitespace and poor formatting remains.

### Expected behavior

I should be able to make the question the size up to the red line and it not cause formatting issues

<img width="1378" alt="Image" src="https://github.com/user-attachments/assets/bd7954fc-b532-40f3-9f3a-53eea8177a67" />

### Information about your Metabase installation

```JSON
{
  "browser-info": {
    "language": "en-US",
    "platform": "MacIntel",
    "userAgent": "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/131.0.0.0 Safari/537.36",
    "vendor": "Google Inc."
  },
  "metabase-info": {
    "databases": [
      "snowflake",
      "postgres"
    ],
    "run-mode": "prod",
    "plan-alias": "pro-cloud",
    "version": {
      "date": "2024-12-16",
      "tag": "v1.52.3.1",
      "hash": "94aa8fd"
    },
    "settings": {
      "report-timezone": "America/Los_Angeles"
    },
    "hosting-env": "unknown",
    "application-database": "postgres"
  }
}
```

### Severity

Blocking usage of the Detail question type

**Comment by harshpatil273** [comment]
I am having similar issues. When the details panel is resized, then the view is compacting, leaving half of the space to the right side. 

![Image](https://github.com/user-attachments/assets/7ed4daad-b50c-4ac3-af6a-4dd2ad097f4f)

Is there something related to how details are visualized. 
Talking in terms of HTML, it seems the default for the visualizing details visualization is col-md-6. Can it be col-md-9/col-md-12 instead?

**Comment by Tony-metabase** [comment]
I am re-opening this since I am not sure why this was closed as not planned. More people are raising interest to use this Viz Type but this format is not ideal for them so opening it to keep it as a reference for these accounts

**Comment by eliotmetabase** [comment]
@Tony-metabase looking at the history, this was discussed here: https://github.com/metabase/metabase/issues/53438

We have looked at fixing this, but it's not straightforward and designs get complex quickly.
I believe we found that the ROI is not worth it, but we can take another swing if we're getting a lot of feedback on it.

**Comment by eliotmetabase** [comment]
Or we could make it a feature request?

**Comment by Tony-metabase** [comment]
@eliotmetabase you're right! I am getting old xD ... and yes make sense! i will close this one and open a separate feature request :)

**Comment by Tony-metabase** [comment]
Changed this to a feature request https://github.com/metabase/metabase/issues/61853
